### DISCUSSION TITLE
HELP

### DISCUSSION THREAD

**Discussion by GENerationXVII** [drizzle-mizzle/CharacterAI-Discord-Bot#75] [Q&A]
It keeps giving me

 /nix/store/4axlc21msjr9g181l2y6hxsmm4jar8i7-dotnet-sdk-6.0.403/sdk/6.0.403/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(144,5): error NETSDK1045: The current .NET SDK does not support targeting .NET 7.0.  Either target .NET 6.0 or lower, or use a version of the .NET SDK that supports .NET 7.0. [/home/runner/CharacterAI-Discord-Bot/CharacterAI_Discord_Bot.csproj]

The build failed. Fix the build errors and run again.

I'm using Replit and I have .NET 7.0.5

**Comment by drizzle-mizzle** [comment]
And what is your question?
What does it has to do with this project? It's not Replit support page.

**Comment by drizzle-mizzle** [comment]
It clearly tells you what is wrong. You use SDK of dotnet 6. You need 7.

**Reply by GENerationXVII** [reply]
Yeah, I am using 7. It's whatever, I will use a different platform to host this.

**Reply by drizzle-mizzle** [reply]
> /nix/store/4axlc21msjr9g181l2y6hxsmm4jar8i7-dotnet-sdk-6.0.403/sdk/6.0.403/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(144,5)

.
> /nix/store/4axlc21msjr9g181l2y6hxsmm4jar8i7-dotnet-sdk-6.0.403/sdk/6.0.403/

.
> dotnet-sdk-6.0.403/sdk/6.0.403

.
> 6.0.403

.
Doesn't look like "7"

**Comment by GENerationXVII** [comment]
Yeah, I do have 7 installed, but it says I'm using 6 💀💀

**Reply by drizzle-mizzle** [reply]
There should be some kind of config where you can change it ig. Don't use replit personally, so I don't know for sure.
